love/src/components/AuthList/AdminAuthList/AdminAuthList.jsx

```diff
@@ -0,0 +1,397 @@
+              Accept
+            </Button>
+            <Button className={styles.actionButton} onClick={() => this.setAuthorization('Reject', row)}>
+              Reject
+            </Button>
+            <Button className={styles.actionButton} onClick={() => this.setAuthorization('More', row)}>
+              More
+            </Button>
+          </>
+        );
+      },
+    },
+  ];
+
+  TABLE_HEADERS = {
+    [AUTHLIST_REQUEST_PENDING]: this.PENDING_HEADERS,
+    [AUTHLIST_REQUEST_ACCEPTED]: this.ACCEPTED_HEADERS,
+    [AUTHLIST_REQUEST_DENIED]: this.REJECTED_HEADERS,
+  };
+
+  handleChange = (option) => this.setState({ showing: option.value });
+
+  handleDateTimeChange = (date, type) => {
+    if (type === 'start') {
+      this.setState({ selectedStartDate: date });
+    } else if (type === 'end') {
+      this.setState({ selectedEndDate: date });
+    }
+  };
+
+  handleMessageChange = (message) => {
+    const trimmedMessage = message.substr(0, MAX_MESSAGE_LEN);
+    this.setState({ message: trimmedMessage !== '' ? trimmedMessage : null });
+  };
+
+  handleDurationChange = (evt) => {
+    const value = evt.target.value;
+    let duration = value === '' || isNaN(value) ? 0 : parseInt(value, 10);
+    if (duration <= MIN_DURATION) duration = MIN_DURATION;
+    else if (duration >= MAX_DURATION) duration = MAX_DURATION;
+    this.setState({ duration: duration !== '' ? duration : null });
+  };
+
+  renderModalFooter = () => {
+    const { authorizationType } = this.state;
+    return (
+      <div className={styles.modalFooter}>
+        <Button
+          className={styles.borderedButton}
+          onClick={() => this.setState({ confirmationModalShown: false })}
+          status="transparent"
+        >
+          Go back
+        </Button>
+        <Button onClick={() => this.authorizeRequest(authorizationType)} status="default">
+          Yes
+        </Button>
+      </div>
+    );
+  };
+
+  componentDidMount() {
+    ManagerInterface.getAuthListRequests().then((res) => {
+      this.setState({
+        authListRequests: res,
+      });
+    });
+
+    ManagerInterface.getTopicData('event-telemetry').then((data) => {
+      const cscList = ['All'].concat(
+        Object.keys(data)
+          .map((x) => `${x}:0`)
+          .sort(),
+      );
+      this.setState({ cscOptions: cscList });
+    });
+
+    // Used to update Table every second and see duration animation
+    if (this.timer) clearInterval(this.timer);
+    this.timer = setInterval(() => {
+      this.setState((prevState) => ({ counter: prevState.counter + 1 }));
+    }, 1000);
+  }
+
+  render() {
+    const {
+      showing,
+      cscOptions,
+      selectedCSC,
+      keywords,
+      confirmationModalShown,
+      confirmationModalText,
+      moreModalShown,
+      message,
+      duration,
+      selectedStartDate,
+      selectedEndDate,
+      selectedRequest,
+      authListRequests,
+    } = this.state;
+
+    // console.log(authListRequests);
```

Comment:
This console.log must be removed

Reply:
I've removed this comment on further commits. Thanks!